extensions/adobe/experience/intelligentServices/profile-journeyai-engagementscores.schema.json

```diff
@@ -0,0 +1,103 @@
+              "type": "object",
+              "isRequired": false,
+              "properties": {
+                "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailEngagementScores": {
+                  "title": "emailEngagementScores",
+                  "description": "Engagement scores for email channel messages",
+                  "type": "object",
+                  "isRequired": false,
+                  "properties": {
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailClickEngagementScore": {
+                      "title": "emailClickEngagementScore",
+                      "description": "probability for a customer to click on links in an email",
+                      "type": "number",
+                      "isRequired": false,
+                      "meta:xdmType": "number"
+                    },
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailOpenEnagementScore": {
+                      "title": "emailOpenEnagementScore",
+                      "description": "probability of a customer to open an email",
+                      "type": "number",
+                      "isRequired": false,
+                      "meta:xdmType": "number"
+                    },
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailRetentionLevel": {
+                      "title": "emailRetentionLevel",
+                      "description": "qualitative categorization of customer's likelihood of staying subscribed to the email list",
+                      "type": "string",
+                      "isRequired": false,
+                      "meta:xdmType": "string"
+                    },
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailRetentionPercentRank": {
+                      "title": "emailRetentionPercentRank",
+                      "description": "percentile rank of the likelihood of staying subscribed to the email list",
+                      "type": "number",
+                      "isRequired": false,
+                      "meta:xdmType": "number"
+                    },
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailUnsubscriptionProbability": {
+                      "title": "emailUnsubscriptionProbability",
+                      "description": "the probability for recipient to unsubscribe from email channel given one email opened",
+                      "type": "number",
+                      "isRequired": false,
+                      "meta:xdmType": "number"
+                    }
+                  },
+                  "meta:xdmType": "object"
+                },
+                "https://ns.adobe.com/experience/intelligentServices/journeyAI/engagementScoresInfo": {
+                  "title": "engagementScoresInfo",
```

Comment:
Engagement Scores Info (No Period "." in the end)

Please change all the titles

Reply:
Done